gms/gossiper.cc

```diff
@@ -974,7 +976,7 @@ void gossiper::run() {
                 do_status_check().get();
             }
 
-            replicate_live_endpoints_on_change();
+            replicate_live_endpoints_on_change().get();
```

Comment:
This is ignoring a future, no? how does it even compile?

Reply:
It's not ignoring it, it's `get()`ting it